migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/DefaultMigrationTarget.java

```diff
@@ -376,6 +378,7 @@ private void saveLibraryChangeLog(Library library, Collection<ChangeLog> changes
   public void saveLibraryDilutions(final Collection<LibraryDilution> libraryDilutions) throws IOException {
     log.info("Migrating library dilutions...");
     for (LibraryDilution ldi : libraryDilutions) {
+      log.debug("Saving library dilution " + ldi.getPreMigrationId());
```

Comment:
preMigrationId is useful when using PinerySource, but may be null if using a MigrationSource other than Pinery (e.g. LoadGenerator), resulting in a lot of "Saving library dilution null."

Maybe use "Saving library dilution of library $libraryAlias" if preMigrationId is null, and "...formerly $preMigrationId" otherwise? Dilutions don't have any good identifier before save, but if there's an error, knowing the Library would help narrow it down. After save, the "formerly ..." bit could be left out if preMigrationId is null.